Kudu.Contracts/Deployment/DeploymentInfoBase.cs

```diff
@@ -76,5 +76,9 @@ public bool IsValid()
         // If DoSyncTriggers is set to true, the after Linux Consumption function app deployment,
         // will initiate a POST request to http://appname.azurewebsites.net/admin/host/synctriggers
         public bool DoSyncTriggers { get; set; }
+
+        // This config is for Linux Consumption function app only
+        // Allow remote build even when WEBSITE_RUN_FROM_PACKAGE is set to a Url (RunFromRemoteZip)
+        public bool ForceRemoteBuild { get; set; }
```

Comment:
This would need to execute the Oryx Build if set to true ? If yes, this could be merged with Deployment Context where we check if build is enabled or not ?

Reply:
@sanchitmehta , This is from the api/zipdeploy?forcetemotebuild parameter. As customer requested, when this parameter is set to true, we will overwrite the WEBSITE_RUN_FROM_PACKAGE instead of SCM_RUN_FROM_PACKAGE after remote build.